apps/daimo-mobile/src/common/nav.ts

```diff
@@ -196,7 +208,7 @@ export function useOnboardingDeepLinkHandler(dc: DaimoChain) {
 export async function handleOnboardingDeepLink(
   dc: DaimoChain,
   nav: OnboardingNav,
-  str: string,
+  str: string
```

Comment:
VSCode > CmdShiftP > Reload Window to get the new typescript

then format document